components/DropdownHeader.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { FunctionComponent, useRef } from "react";
+import HeaderItems from './HeaderItems';
+import Image from "next/image";
+import Icon from "@mdi/react";
+import { mdiClose } from "@mdi/js";
+
+const menuItems = [
+  {
+    title: 'Quem Somos',
+    url: '#',
+    submenu: [
+      {
+        title: 'Nosso time',
+        url: '#',
+      },
+    ],
+  },
+  {
+    title: 'Soluções',
+    url: '#services',
+    submenu: [
+      {
+        title: 'Aplicações personalizadas',
+        url: '#',
+      },
+      {
+        title: 'Consultoria em desenvolvimento',
+        url: '#',
+      },
+      {
+        title: 'Soluções Sharpy',
+        url: '#',
+      },
+    ],
+  },
+  {
+    title: 'Carreiras',
+    url: '#',
+  },
+  {
+    title: 'Fale Consco',
+    url: '#',
+  },
+]
+
+interface Props {

```

Comment:
Interfaces should be in the types folder and it is a good practice to use intuitive names for the interface like IDropdownHeader. The name of the type in types pages can be DropdownHeaderProps.ts